stdlib/event/event.lua

```diff
@@ -4,8 +4,25 @@
 require 'stdlib/core'
 require 'stdlib/game'
 
+core_events = { init = -1, load = -2, configuration_changed= -3 }
+
 Event = {_registry = {}}
 
+script.on_init(function()
```

Comment:
I like the idea of adding the core events to the registry, but this presents a problem as currently implemented. Say a mod is currently using the Event registry, and then requires 'stdlib/event/event', won't these functions override what a mod might already have set as the handler?

I think we need to add the script.on_init, on_load, and on_configuration_changed in the Event.register call, when a core event is registered, so we know we are setting the handler for a mod that explicitly requested a core handler.